build.gradle

```diff
@@ -25,9 +25,12 @@ dependencies {
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     }
-    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-    implementation('org.postgresql:postgresql')
     implementation 'org.liquibase:liquibase-core'
+    implementation 'org.mongodb:mongo-java-driver'
+    implementation 'org.springframework.data:spring-data-mongodb'
+    implementation 'org.springframework.amqp:spring-rabbit'
+    compile group: 'com.google.guava', name: 'guava', version: '29.0-jre'
```

Comment:
do you really need guava? :)

Reply:
I use this dependency to get ImmutableMap

Reply:
guava is really heavy dependency... it isn't worth having it only for ImmutableMap
Please, read once again about Java 9-14 features:
https://www.marcobehler.com/guides/a-guide-to-java-versions-and-features
you will find `Map.of()` function to be pretty similar to `ImmutableMap.of()`

Reply:
😳🙈